.gitmodules

```diff
@@ -1,7 +1,6 @@
 [submodule "acquire_zarr/external/vcpkg"]
 	path = acquire_zarr/external/vcpkg
 	url = https://github.com/microsoft/vcpkg.git
-[submodule "acquire_zarr/external/acquire-driver-zarr"]
-	path = acquire_zarr/external/acquire-driver-zarr
-	url = https://github.com/acquire-project/acquire-driver-zarr.git
-	branch = afa9ace
+[submodule "acquire_zarr/external/acquire-zarr-library"]
+	path = acquire_zarr/external/acquire-zarr-library
```

Comment:
Is this because your ROS package is named `acquire_zarr`?

Reply:
Yeah,  I wanted it to have a different name (at least within the ROS context) than the node.